client/src/components/OnboardingOne/OnboardingOne.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import "./OnboardingOne.css";
+import InputTextField from "../InputTextField/InputTextField.js";
+import Button from "../Button/Button.js";
+import OnboardingParent from "../OnboardingParent/OnboardingParent";
+
+class OnboardingOne extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    this.state = {
+      classList: "OnboardingOne"
+    };
+  }
+
+  render() {
+    return (
+      <OnboardingParent>
+        <div className={this.state.classList}>
```

Comment:
If you're not updating state in this component, does it need to be a class component? Do you need to set the class from state?